engine/jobmaster/dm/dm_jobmaster.go

```diff
@@ -161,6 +162,7 @@ func (jm *JobMaster) handleOnlineStatus(worker framework.WorkerHandle) error {
 		return err
 	}
 
+	jm.finishedStatus.Delete(taskStatus.Task)
```

Comment:
in which case, will worker become Offine with Finished then become Online?

Reply:
when switch to another unit.